tox.ini

```diff
@@ -139,3 +139,10 @@ commands_pre =
     --compile-bytecode
 commands =
   uv tool run arize-phoenix {posargs:serve}
+
+[testenv:ruff]
```

Comment:
```suggestion
[testenv:format]
```

Reply:
it's also got linting though

Reply:
non-blocking

Reply:
e.g. [this](https://docs.astral.sh/ruff/rules/numpy2-deprecation/) would be lint, and not format